bundles/org.openhab.automation.luascripting/pom.xml

```diff
@@ -0,0 +1,52 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <groupId>org.openhab.addons.bundles</groupId>
+    <artifactId>org.openhab.addons.reactor.bundles</artifactId>
+    <version>3.1.0-SNAPSHOT</version>
+  </parent>
+
+  <artifactId>org.openhab.automation.luascripting</artifactId>
+
+  <name>openHAB Add-ons :: Automation :: Lua Scripting</name>
+
+  <properties>
+    <bnd.fixupmessages><![CDATA["Classes found in the wrong directory","The default package '.' is not permitted by the Import-Package syntax"; restrict:=error; is:=warning]]></bnd.fixupmessages>
+  </properties>
+
+  <!-- For whatever reason the luaj developers never released their latest version on Maven Central, once they do this can
+    be removed -->
```

Comment:
Perhaps @J-N-K can add the JARs to the openHAB Maven repository so we don't need to store them in the repo?

Reply:
Ooops. Didn't see that you mentioned me here. Of course I can do that.